giri/parser.js

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
+
+'use strict';
+const Parser = require('../.lib/parser.js');
+
+/**
+ * Recognizes the accesses to the platform Giri Repbase
+ * @param  {Object} parsedUrl an object representing the URL to analyze
+ *                            main attributes: pathname, query, hostname
+ * @param  {Object} ec        an object representing the EC whose URL is being analyzed
+ * @return {Object} the result
+ */
+module.exports = new Parser(function analyseEC(parsedUrl, ec) {
+  let result = {};
+  let path = parsedUrl.pathname;
+  // uncomment this line if you need parameters
+  // let param = parsedUrl.query || {};
+
+  // use console.error for debuging
+  // console.error(parsedUrl);
+
+
+  if (/^\/repbase\/update\/search.php$/i.test(path)) {
```

Comment:
The `dot` is a special character, and therefore must be escaped

```suggestion
  if (/^\/repbase\/update\/search\.php$/i.test(path)) {
```